script/deploy/mainnet/012_xOGNGovernanceScript.sol

```diff
@@ -47,45 +39,57 @@ contract XOGNGovernanceScript is BaseMainnetScript {
 
         _recordDeploy("XOGN_GOV", address(governance));
 
-        _buildGnosisTx();
+        _buildGovernanceProposal();
     }
 
-    function _buildGnosisTx() internal {
+    function _buildGovernanceProposal() internal {
         Timelock timelock = Timelock(payable(Addresses.TIMELOCK));
 
         address xognGov = deployedContracts["XOGN_GOV"];
 
-        govFive.setName("Enable OGN Governance & Begin Rewards");
+        address ognRewardsSourceProxy = deployedContracts["OGN_REWARDS_SOURCE"];
+        address veOgvImpl = deployedContracts["VEOGV_IMPL"];
 
-        govFive.setDescription("Grant roles on Timelock to OGN Governance");
+        govProposal.setName("Grant access to OGN Governance");
 
-        govFive.action(Addresses.TIMELOCK, "grantRole(bytes32,address)", abi.encode(timelock.PROPOSER_ROLE(), xognGov));
-        govFive.action(Addresses.TIMELOCK, "grantRole(bytes32,address)", abi.encode(timelock.CANCELLER_ROLE(), xognGov));
-        govFive.action(Addresses.TIMELOCK, "grantRole(bytes32,address)", abi.encode(timelock.EXECUTOR_ROLE(), xognGov));
+        govProposal.setDescription("Grant access to OGN Governance");
 
-        // Enable rewards for staking
-        govFive.action(
-            deployedContracts["OGN_REWARDS_SOURCE"],
-            "setRewardsPerSecond(uint192)",
-            abi.encode(uint192(REWARDS_PER_SECOND))
+        // Grant access to OGN Governance
+        govProposal.action(
+            Addresses.TIMELOCK, "grantRole(bytes32,address)", abi.encode(timelock.PROPOSER_ROLE(), xognGov)
+        );
+        govProposal.action(
+            Addresses.TIMELOCK, "grantRole(bytes32,address)", abi.encode(timelock.CANCELLER_ROLE(), xognGov)
+        );
+        govProposal.action(
+            Addresses.TIMELOCK, "grantRole(bytes32,address)", abi.encode(timelock.EXECUTOR_ROLE(), xognGov)
         );
 
-        if (!isForked) {
-            govFive.printTxData();
-        }
+        // Revoke access from Multisig
```

Comment:
Let's keep an overlapping period and not revoke the multisig until after we've run a test TX through the final governance. This should break out into a new deploy file.

Reply:
This file has a skip method that doesn't run it on mainnet

Reply:
We still need to separate the granting gov time permissions into a separate file from revoking multi-sig timelock permissions (even though both should not run now)

Reply:
Ah, gotcha, will do that